soc/xtensa/intel_apl_adsp/CMakeLists.txt

```diff
@@ -7,3 +7,5 @@ zephyr_library_sources(soc.c)
 zephyr_library_sources(main_entry.S)
 
 zephyr_library_sources_ifdef(CONFIG_SMP soc_mp.c)
+
+add_subdirectory(common)
```

Comment:
what is that? I do not see you adding anything into it.

Reply:
Sorry, this was the result of cherry-pick from https://github.com/thesofproject/zephyr/commit/25812de5c1503031d3fb54e8e5fccb1ab3149041
Fixed now.